core/testMinimumPhpVersion.php

```diff
@@ -200,6 +200,7 @@ function Piwik_GetErrorMessagePage(
                                 Please click below if you are not redirected in ' . $countdown . ' seconds</br></br>
                                 Go to <a href="' . $redirectUrl . '">' . $redirectUrl . '</a> 
```

Comment:
to avoid any possible xss or similar we should maybe escape `$redirectUrl`

Reply:
@sgiehl do you know what the best way to escape is here considering we're in `testMinimumPhpVersion` that should potentially work with any PHP version and we don't have twig here? Or maybe this part wouldn't need to be compatible with older PHP versions and we could potentially load some escaping methods?